### ISSUE TITLE
[enhancement] Add TV Tab to Quick Create Resource Modal form

### ISSUE THREAD

**Issue by Ruslan-Aleev** [modxcms/revolution#14270]
### Summary
It is necessary to add the TV selection tab to the form for quick resource creation, see the picture below.

![res_1](https://user-images.githubusercontent.com/12523676/51084144-8fd4a600-173e-11e9-89ea-354469e8ec0c.png)

p.s. Forms of quick creation of resource now do not fully perform their functions, since after quickly creating resources, you still have to edit them in the usual way.

### Related issue(s)/PR(s)
https://github.com/modxcms/revolution/issues/2580
https://github.com/modxcms/revolution/issues/14267
https://github.com/modxcms/revolution/issues/14266

**Comment by JoshuaLuckers** [comment]
Isn't the purpose of the quick create buttons to quickly create a resource? This way it starts to look like the same as when you create a resource the "normal" way.

**Comment by Ruslan-Aleev** [comment]
Yes, it makes sense :) However, if you cannot completely create a resource with all the elements (TV) and then you have to edit the resources in a normal way - why do you need a form for quickly creating a resource?
I use this quick form when editing other elements, for example, chunks. And, if the resource can be created immediately, it will be much more convenient.
I do not know how complicated the implementation of such tabs is, if it is not very difficult technically - why not to do this? :)

**Comment by ghost** [comment]
TVs were never intended to be used as extended resource fields, they are, after all, _Template_ variables. I think this differentiation will become more pronounced as features for creating custom resources with their own extended fields in extended tables gets simplified.

**Comment by Ruslan-Aleev** [comment]
@sottwell Here I do not agree, TV is associated with the Template, Template with the Resource, therefore, TV is associated with the Resource, though not directly.
And, as it seems to me, TV is the resource fields, they even have the same syntax - [[*longtitle]], [[*menuindex]] and **[[*TVname]]**.

**Comment by alroniks** [comment]
For now, it is technically difficult to implement because when template changes tab with TVs should be updated by fields for the new template. Doing it on the fly is not easy. Even on the normal resource page changing the template requires the reload of the page to apply new tv set. So the idea is not bad but I am not sure that it is possible to implement it on the current js framework.

**Comment by Jako** [comment]
The biggest issue would be custom template variables, that need their own javascripts. That part has to be loaded during changing the template. With the use of the current implementation, this should be quite impossible.
